File: baekjoon/bronze/level1/test_2810/java/Main.java
package bronze.level1.test_2810.java;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = Integer.parseInt(sc.nextLine());
        char[] seats = sc.nextLine().toCharArray();

        int cupCount = 1;
        int personCount = 0;
        for (int i = 0; i < n; i++) {
            if (seats[i] == 'S') {
                cupCount++;
                personCount++;
            } else {
                personCount+=2;
                i++;
                cupCount++;
            }
        }

        System.out.print(Math.min(cupCount, personCount));
    }
}

File: baekjoon/bronze/level1/test_2810/kotlin/Main.kt
package bronze.level1.test_2810.kotlin

fun main() {
    val n = readln().toInt()
    val seats = readln().toCharArray()

    var cupCount = 1;
    var personCount = 0;

    var i = 0
    while (i < n) {
        if (seats[i] == 'S') {
            cupCount++
            personCount++
        } else {
            cupCount++
            personCount += 2
            i++
        }
        i++
    }
    print(cupCount.coerceAtMost(personCount))
}
